pallets/identity-management-mock/src/lib.rs

```diff
@@ -285,15 +285,15 @@ pub mod pallet {
 
 		/// Link an identity
```

Comment:
Can you adjust the comment too? :) (and other places)

Reply:
😢